Towny/src/main/java/com/palmergames/bukkit/towny/object/economy/Account.java

```diff
@@ -219,6 +223,13 @@ public void removeAccount() {
 		TownyEconomyHandler.removeAccount(getName());
 	}
 
+	/**
+	 * @return the townyObject that this Account represents.
+	 */
+	public TownyObject getTownyObject() {
```

Comment:
I think it would be better to make this an abstract method that returns an EconomyHandler, that way the BankAccount class can keep its government field without having to do casts

Reply:
This has been done, but an abstract method wasn't required, as the EconomyHandler is required to make an Account/BankAccount anyways. 

The BankAccount has had its references to the getTownyObject()/getEconomyHandler() returned to the reference to the government field.